File: server/Src/Services/Open/OpenVN.Domain/Entities/Feedback/Feedback.cs
namespace OpenVN.Domain
{
    [Table("feedback")]
    public class Feedback : PersonalizedEntity
    {
        public long ParentId { get; set; }

        public string Content { get; set; }

        public string Path { get; set; }

        public string FromIP { get; set; }
    }
}

File: client/src/app/models/feedback/feedback.ts
import { BaseModel } from "../base/base-model";

export class Feedback extends BaseModel {
  public avatarUrl?= "";
  public email = "";
  public content = "";
  public parentId = "0";
  public hasReply = false;
  public replyCount?= 0;
  public timeFormat = "";
  public fromIP = "";
}
